### DISCUSSION TITLE
On Roadmaps and others

### DISCUSSION THREAD

**Discussion by rishavs** [aardappel/lobster#281] [General]
Hi. Just stumbled upon lobster.

Had a couple of questions;
1. Is there anywhere one can keep track of the Lobster Roadmap and what features are currently being worked upon
2. Any plans on adding a http server into lobster?
3. Any plans about having a package manager in lobster?

Thanks!

**Comment by aardappel** [comment]
1. The closest to this is `TODO.txt` in the repo, which has features more likely to worked on first closer to the top. But there is no real roadmap, I work on Lobster when I have time or feel the need, and it usually goes in bursts when I am inspired to do something.
2. I've looked into this a couple of times already, since it would be useful.. http client also. It all depends on finding a good C++ library for it, one that is very small, simple, yet well tested and has full functionality. Everything I've found so far is gigantic, and/or has large dependencies. Since we build on 6 platforms, besides portability, easy of integration into 6 build systems is a major consideration.
3. I will add this once there is a clear need for it. So far, there is close to no usage of external Lobster packages, people tend to use what is in the main repo and their own self contained codebase. If something is super useful, I so far just add it to the main repo.
